src/main/scripts/run_e2e_test.py

```diff
@@ -103,8 +104,22 @@ def run_shell_command(cmd):
 
 if module_to_build:
     os.chdir("../../..")
+    if module_to_build == 'postgresql-plugin' or module_to_build == 'mssql-plugin' or module_to_build == 'mysql-plugin' or module_to_build == 'oracle-plugin':
```

Comment:
Can we get the list of modules as well from yaml? 
something like `if module_to_build in driver_details['modules']:`

Reply:
Done.